onnx/backend/test/case/node/mod.py

```diff
@@ -53,10 +69,123 @@ def export_int64_mixed_sign():  # type: () -> None
-               name='test_mod_int64_mixed_sign_example')
+               name='test_mod_mixed_sign_int64')
+
+    @staticmethod
+    def export_mod_mixed_sign_int32():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([-4, 7, 5, 4, -7, 8]).astype(np.int32)
+        y = np.array([2, -3, 8, -2, 3, 5]).astype(np.int32)
+        z = np.mod(x, y)  # expected output [ 0, -2,  5,  0,  2,  3]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_mixed_sign_int32')
+
+    @staticmethod
+    def export_mod_mixed_sign_int16():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([-4, 7, 5, 4, -7, 8]).astype(np.int16)
+        y = np.array([2, -3, 8, -2, 3, 5]).astype(np.int16)
+        z = np.mod(x, y)  # expected output [ 0, -2,  5,  0,  2,  3]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_mixed_sign_int16')
+
+    @staticmethod
+    def export_mod_mixed_sign_int8():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([-4, 7, 5, 4, -7, 8]).astype(np.int8)
+        y = np.array([2, -3, 8, -2, 3, 5]).astype(np.int8)
+        z = np.mod(x, y)  # expected output [ 0, -2,  5,  0,  2,  3]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_mixed_sign_int8')
+
+    @staticmethod
+    def export_mod_uint8():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([4, 7, 5]).astype(np.uint8)
+        y = np.array([2, 3, 8]).astype(np.uint8)
+        z = np.mod(x, y)  # expected output [0, 1, 5]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_uint8')
+
+    @staticmethod
+    def export_mod_uint16():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([4, 7, 5]).astype(np.uint16)
+        y = np.array([2, 3, 8]).astype(np.uint16)
+        z = np.mod(x, y)  # expected output [0, 1, 5]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_uint16')
+
+    @staticmethod
+    def export_mod_uint32():  # type: () -> None
+        node = onnx.helper.make_node(
+            'Mod',
+            inputs=['x', 'y'],
+            outputs=['z'],
+        )
+
+        x = np.array([4, 7, 5]).astype(np.uint32)
+        y = np.array([2, 3, 8]).astype(np.uint32)
+        z = np.mod(x, y)  # expected output [0, 1, 5]
+        expect(node, inputs=[x, y], outputs=[z],
+               name='test_mod_uint32')
+               
```

Comment:
The Travis builds fail here with the following message:
+flake8
./onnx/backend/test/case/node/mod.py:157:1: W293 blank line contains whitespace